AVL_tree/AVL_tree/Class1.cs

```diff
@@ -0,0 +1,274 @@
+﻿using System;
+
+namespace AVL_tree
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            AVL<int, string> tree = new AVL<int, string>();
+            tree.Add(5, "a");
+            tree.Add(3, "b");
+            tree.Add(7, "c");
+            tree.Add(2, "d");
+            tree.Add(4, "e");
+            tree.Add(9, "j");
+            tree.Delete(7);
+            tree.DisplayTree();
+        }
+    }
+    class AVL<TKey, TValue> where TKey : System.IComparable, System.IEquatable<TKey>
+    {
+        class Node
```

Comment:
Nested classes - not good practice. I think be better add new file for this class too